semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -174,6 +176,24 @@ def to_markdown_attr(
             )
         )
 
+    @staticmethod
+    def to_markdown_metric_table(
+        semconv: MetricSemanticConvention, output: io.StringIO
+    ):
+        """
+        This method renders metrics as markdown table entry
+        """
+        output.write(
+            "| Name     | Instrument       | Unit ([UCUM](README.md#instrument-units)) | Description    |\n"
```

Comment:
This relative link URL won't work if there is no README.md in the same folder. This is currently a limitation of this tool (#20). I recommend rather not using a link, or maybe using a absolute link (having the drawback that you need to choose a branch).

Reply:
I was trying to regenerate what's already in the semantic convention markdown currently. 

If we need to remove the link in order to get it to work properly with the tool, I'm fine with that. So I've gone ahead and removed the link.